app/controllers/form_controller.rb

```diff
@@ -116,7 +120,7 @@ def restore_previous_errors(previous_errors)
     return unless previous_errors
 
     previous_errors.each do |attribute, message|
-      @log.errors.add attribute, message.first
+      @log.errors.add attribute, message.html_safe
     end
   end
```

Comment:
What the change here for?

Reply:
Hmm, I think cause it's now coming through as a plain string as opposed to part of error object, it must have been handled within the error before in the individual cases we were setting it as html_safe.

I don't _think_ we're playing back any content that could mean this is harmful in our errors, and I think we already apply html_safe to all our error messages in BU error list, but please let me know if you can think of places where this could cause issues!